documents/reviews/CodeReview.tex

```diff
@@ -0,0 +1,49 @@
+\documentclass{article}
+
+\title{Document Review}
+\author{Reviewer: name Reviewee: name }
+\date{\today}
+
+\begin{document}
+
+\maketitle
+
+\section{scoring}
+    Provide a rating from of 1 - 5. \\
+    1 - Work was incomplete or extremely poorly done\\
+    3 - Work was done to an acceptable standard\\
+    5 - The work went above and beyond the necessary components\\
+
+\begin{tabular}{|l|c|}
+	\hline
+	Criteria & Score\\
+	\hline
+    Completeness:
+    (completed to the full scope of the requirements) & \\
+	\hline
+    Concise:
+    (the purpose is achieved without undue complication) & \\
+	\hline
+    Documentation (Including author): & \\
+    \hline
+    Structure/organization:
+    (does the work flow, are components grouped logically?) & \\
+    \hline
+    Syntax: & \\
+    \hline
+    Modulatity: & \\
+    \hline
+    Naming Conventions: & \\
```

Comment:
```suggestion
    Adheres to Coding Conventions: & \\
```
I think this should be broadened. Just following naming conventions is a small part of our code style